app/controllers/articles_controller.rb

```diff
@@ -0,0 +1,5 @@
+class ArticlesController < ApplicationController
+  def index
+    @pages = Page.published.order('updated_at DESC')
```

Comment:
Since pages get updated on every change (even updates to some associations, e.g. plugins or images), I'd recommend we sort by `created_at` instead.